WordPress/Classes/ViewRelated/Gutenberg/Processors/GutenbergGalleryUploadProcessor.swift

```diff
@@ -28,76 +28,67 @@ class GutenbergGalleryUploadProcessor: Processor {
         static let dataLink = "data-link"
     }
 
-    lazy var imgPostMediaUploadProcessor = HTMLProcessor(for: ImageKeys.name, replacer: { (img) in
-        guard let imgClassAttributeValue = img.attributes[ImageKeys.classAttributes]?.value,
-            case let .string(imgClass) = imgClassAttributeValue else {
-                return nil
-        }
+    func processImgPostMediaUpload(_ element: Element) {
+        let imgTags = try? element.select(ImageKeys.name)
+        imgTags?.forEach {imgTag in
```

Comment:
Optional nit: Could be in an `if let`, or could `guard` and return early to remove the optional.

Reply:
I applied this suggestion in https://github.com/wordpress-mobile/WordPress-iOS/pull/22896/commits/d60c1d19412f8df4723a174ab3982ea0af4fe6f1.